docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+web:
+  build: .
+  ports:
+    - "4000:4000"
+  command: mix phoenix.server
+  environment:
+    - MIX_ENV=dev
+    - PORT=4000
+  volumes:
+    - .:/app
+  working_dir: /app
+  links:
+    - db
+db:
+  image: postgres:9.5
+  environment:
+    - POSTGRES_USER=postgres
```

Comment:
Yeah this is the problem.  It cannot find the environment variables. If we are setting up the variables in test setup then it needs to be DB_ENV_POSTGRES_USER, DB_ENV_POSTGRES_PASSWORD.